do_freebsd.sh

```diff
@@ -1,20 +1,48 @@
 #!/bin/sh -xve
 NPROC=`sysctl -n hw.ncpu`
 
+# we need bash first otherwise almost nothing will work
+if [ ! -L /bin/bash ]; then
+    echo install bash and link /bin/bash to /usr/local/bin/bash
+    echo Run:
+    echo     sudo pkg install bash
+    echo     ln -s /usr/local/bin/bash /bin/bash
+    exit 1
+fi
 if [ x"$1"x = x"--deps"x ]; then
-    # we need bash first otherwise almost nothing will work
-    sudo pkg install bash
-    if [ ! -L /bin/bash ]; then
-        echo linking /bin/bash to /usr/local/bin/bash
-        ln -s /usr/local/bin/bash /bin/bash
-    fi
     sudo ./install-deps.sh
 fi
+if ! grep -q ENODATA /usr/include/errno.h; then
+    echo Need ENODATA in /usr/include/errno.h for cython compilations
+    echo Please add it manually after ENOATTR with value 87
+    echo '#define ENOATTR         87'
```

Comment:
@tchaikov 
Given the fact that we need an upstream accept from Cython (Which I already submitted) and again downstream into the FreeBSD ports, is it wise to keep the warning for the time being.